### PR TITLE
feat: landing page

### LINKED ISSUE #17: Feature: Landing Page
**Describe the new feature**
Let's add the following contents to the landing page (`unixsocial.com/`):

1. Right top corner has an [X icon](https://lucide.dev/icons/x) which does a redirect to **Blog page** on click
2. Logo in the middle _(add to the `asset/images` or `public/images` folder which is part of prod)_
3. Large primary button with text: "Check it out" which does a redirect to **Blog page** on click
4. Wide Radix Wallet Connect button (use TailwindCSS css variable set: `[--radix-connect-button-width:100px]`)
5. Add the social icons and open each social platform in a new tab using links defined in the `README.md`
_(lets discuss how we got those logos as [Lucide React Icons](https://lucide.dev/) has not supported all of them)_
   1. X
   2. Telegram
   3. Discord
   4. Tik-tok
   6. Instagram

**Where the feature should be placed**
under the base url `/` within landing page

**Sketches**
Video description has been provided via Telegram

**Additional context**
Has dependency on #13 #24

### PR DESCRIPTION
**Pull request by AhsenRiaz**
fix #17

### CODE REVIEW

**Comment by attilacsanyi** [comment]
Can we change social icons to render similar? As you can see discord logo has different colour. Thx @AhsenRiaz 
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 117.0.0.0 on Android 10
Viewport:  424 x 845 @2.549999952316284x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-42--unix-web.netlify.app%2F&os=Android&os_version=10&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-42--unix-web.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=65087ee1dfc5fe0008990c46&commentID=65088367a043592330681261&resolution=resolved)

**Comment by AhsenRiaz** [comment]
@attilacsanyi I have set different colors on purpose so that @uniscinft can decide which colors seem suitable for the UI. Let me know what you think about this and I'll make them the same then.

**Comment by attilacsanyi** [comment]
Cool ask then @uniscinft :) Our theme has already the primary color like the "Check it out", how would it look? If we add a new color then would be great to add to the `storybook` design and css variables. Thanks.

**Comment by uniscinft** [comment]
Looks good, we can keep the different colors for the signs. Great job. Is it possible to make the logo and buttons bigger?

**Comment by attilacsanyi** [comment]
Different colours are not intentional afaik @uniscinft should align with our theme, but I am fine if you pick one shade only. @AhsenRiaz do you also think we need multiple colours? 🤔

**Comment by AhsenRiaz** [comment]
@attilacsanyi I think selecting only 1 shade would be the better approach. I have used 5 shades so that  @uniscinft can let us know which 1 shade looks most good and then I can make them all the same as per the selected shade.

**Comment by AhsenRiaz** [comment]
@uniscinft Yes it's possible to make the logo and buttons bigger. I'll make them big in the next iteration.

File: libs/ui/src/lib/radix-connect/radix-connect.tsx [hunk 1]
```diff
@@ -12,7 +12,9 @@ declare global {
 
 export const RadixConnect = () => {
   return (
-    <span className="inline-flex [--radix-connect-button-width:100px] sm:[--radix-connect-button-width:138px]">
+    <span
```
Comment (attilacsanyi) [inline libs/ui/src/lib/radix-connect/radix-connect.tsx:15]: Are you sure it is supporting mobile view on the blog page? Below 138px as radix documentation stated the login button will shrink and needed. Radix connect might be different on Blog page 🤔

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 2]
```diff
@@ -1,40 +1,71 @@
-import { Theme, useTheme } from '@unix/feature/core';
-import { Button, RadixConnect, ThemeToggle } from '@unix/ui';
-import { X } from 'lucide-react';
+import { Button, RadixConnect } from '@unix/ui';
+import Logo from 'public/unix-logo.png';
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:2]: Can we use `webp` version here as well?

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 3]
```diff
@@ -1,40 +1,71 @@
...
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img src={Discord} alt="Discord" />
+            </a>
+          </Button>
+          <Button variant={'link'} size={'icon'}>
+            <a
+              href="https://www.tiktok.com/@unixsocial?_t=8fAUGTeqXQG"
+              target="_blank"
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:52]: `rel="noopener noreferrer"` is missing here, how about defining a `socialLinks` array variable and iterate over, so no repetition needed.

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 4]
```diff
@@ -1,40 +1,71 @@
...
+            >
+              <img src={TikTok} alt="TikTok" />
+            </a>
+          </Button>
+          <Button variant={'link'} size={'icon'}>
+            <a href="https://twitter.com/unixsocial" target="_blank">
+              <img src={X} alt="Twitter" />
+            </a>
+          </Button>
+          <Button variant={'link'} size={'icon'}>
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:62]: `size='icon'` is simpler here :)

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 5]
```diff
@@ -1,40 +1,71 @@
...
+            <a
+              href="https://www.tiktok.com/@unixsocial?_t=8fAUGTeqXQG"
+              target="_blank"
+            >
+              <img src={TikTok} alt="TikTok" />
+            </a>
+          </Button>
+          <Button variant={'link'} size={'icon'}>
+            <a href="https://twitter.com/unixsocial" target="_blank">
+              <img src={X} alt="Twitter" />
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:59]: Please add `alt` and `aria-label` to satisfy accessibility

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 6]
```diff
@@ -1,40 +1,71 @@
...
-      <main className="flex-grow">
-        <section className="relative">
-          <div className="mx-auto max-w-6xl px-4 sm:px-6">
-            {/*  Main content */}
-          </div>
-        </section>
-      </main>
-    </>
+        <div className="pb-4">
+          <Button asChild size={'lg'}>
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:24]: It seems we need to [update the light primary color for better acessibility](https://app.netlify.com/sites/unix-web/deploys/6509b3e9db1ae4000844434f).

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 7]
```diff
@@ -1,40 +1,82 @@
...
-          <div className="flex h-16 items-center justify-between md:h-20">
-            {/* Unix Logo */}
-            <div>Logo</div>
-
-            <div className="flex items-center justify-between gap-2">
-              <RadixConnect />
-              <ThemeToggle onChangeTheme={theme => setTheme(theme as Theme)} />
+  const socialMediaBtns = [
+    {
+      ariaLabel: 'Instagram icon btn',
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:13]: [Aria is for screen readers](https://www.aditus.io/aria/aria-label/) so use just: "Instagram"

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 8]
```diff
@@ -1,40 +1,82 @@
...
+          <img
+            width="160px"
+            height="160px"
+            className="mx-auto"
+            src={Logo}
+            alt="unix-logo"
+          />
+        </div>
+        <div className="pb-4">
+          <Button aria-label="Check out blog" asChild size="lg">
```
[the remark this answers is not in the document]
Comment (AhsenRiaz) [reply libs/feature/landing/src/lib/landing-page.tsx:57]: Should I change it to 'Check Out'?

[the remark this answers is not in the document]
Comment (attilacsanyi) [reply libs/feature/landing/src/lib/landing-page.tsx:57]: yeah that wil ldo the job :)

File: libs/feature/landing/src/lib/landing-page.tsx [hunk 9]
```diff
@@ -1,40 +1,82 @@
...
-        <section className="relative">
-          <div className="mx-auto max-w-6xl px-4 sm:px-6">
-            {/*  Main content */}
-          </div>
-        </section>
-      </main>
-    </>
+        <div className="flex w-80 justify-between">
+          {socialMediaBtns.map((smb, i) => (
+            <Button
```
Comment (attilacsanyi) [inline libs/feature/landing/src/lib/landing-page.tsx:67]: I recommend this snippet to show icons are clickable, also provided `w` and `h` for logos:
```jsx
            <Button
              key={i}
              aria-label={smb.ariaLabel}
              variant="ghost"
              size="icon"
            >
              <a href={smb.href} target="_blank" rel="noopener noreferrer">
                <img className="h-6 w-6" src={smb.src} alt={smb.alt} />
              </a>
            </Button>
```

Comment (attilacsanyi) [reply libs/feature/landing/src/lib/landing-page.tsx:67]: Even would be better not adding fixed bg color for the images as we cannot style to our theme, you can see if `variant="ghost"` is removed so it means we use our primary theme.

**Comment by attilacsanyi** [comment]
Be sure to check in mobile view. In this case Radix Connect is becoming small button. We might need to add props to it which influence the width of the button depending whether we need mobile small version or not  On blog we might need to shrink as now to fit into the header.
<details><summary>Browser metadata</summary>





```
Path:      /
Browser:   Chrome 117.0.0.0 on Android 10
Viewport:  424 x 845 @2.549999952316284x
Language:  en-GB
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-42--unix-web.netlify.app%2F&os=Android&os_version=10&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-42--unix-web.netlify.app/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=650b208bee5e360008cf607d&commentID=650b4684ca35013afea94974&resolution=resolved)

**Review by attilacsanyi — APPROVED** [review]
Looks good. You can merge if you also like it.

**Comment by AhsenRiaz** [comment]
Nice work @attilacsanyi . UI looks more clean now.

**Pull request merged**